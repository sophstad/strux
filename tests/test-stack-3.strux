int main() {
	Stack::string:: s = new Stack::string::();
	s.add("S");
	s.add("OOOO");
	s.add("WWWW");
	s.pop();
	string topelement = s.peek();
	int x = s.size();
	print(x);
	print(topelement);
	return 0;
}