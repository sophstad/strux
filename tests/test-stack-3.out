2
OOOO
Stack is empty!
